### PR TITLE
Bump Windows Py3 builds timeout to 10 hours

### PR DESCRIPTION
**Pull request by s0undt3ch**
### What does this PR do?
See title.

### What issues does this PR fix or reference?
https://github.com/saltstack/salt/issues/56234

### CODE REVIEW

File: .ci/kitchen-windows2016-py3 [hunk 1]
```diff
@@ -10,7 +10,7 @@ runTestSuite(
     nox_env_name: 'runtests-zeromq',
     nox_passthrough_opts: '--unit',
     python_version: 'py3',
-    testrun_timeout: 9,
+    testrun_timeout: 10,
```
Comment (Ch3LL) [inline .ci/kitchen-windows2016-py3]: windows 2019 is the only one that is consitantly timing out, not 2016

**Review by Ch3LL — CHANGES_REQUESTED** [review]

File: .ci/kitchen-windows2016-py3 (hunk 1 above)
Comment (s0undt3ch) [reply .ci/kitchen-windows2016-py3]: Ok, I'll reduce the scope of the change.

Comment (s0undt3ch) [reply .ci/kitchen-windows2016-py3]: Done

**Comment by bryceml** [comment]
this doesn't affect 2016?

**Review by bryceml — APPROVED** [review]

**Comment by bryceml** [comment]
> this doesn't affect 2016?

oh, I read megan's comment, sounds good.

**Review by Ch3LL — APPROVED** [review]

**Review by waynew — APPROVED** [review]

**Review by twangboy — APPROVED** [review]

**Pull request merged**